wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+           financial_sector == "Other" &
+           investor_name != "Meta Investor") %>% 
+  select(sectors_with_assets, company_name, market_value, value_usd) %>% 
+  group_by(sectors_with_assets, company_name) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(share_total_value = round(
+    sum_value_usd/(total_overview %>% filter(asset_type=="Bonds") %>% pull(sum_value_usd)),6)) %>% 
+  arrange(sectors_with_assets, desc(share_total_value))
+datatable(non_pacta_ALD_CB_detail)
+
+
+```
+
+
+## Funds Coverage
+Check the impact of funds on overall coverage.
+Aggregation should happen on value_usd level where possible to ensure comparability across funds regarding the impact on the overall PF value
+For cases with missing currency information, we add aggregation by market_value, and group by currency.
+
+1. List of fund ISINs and their Values impact on the overall PF value, filtered by direct_holding == FALSE and asset_type == "Funds"
+2. List of fund_isins that have ISIN == "MissingValue" and are therefore omitted by the first table. This, however, seems to come from the normalise_fund_data() function and may not be necessary. It is introduced on purpose for fund holdings, whose overall sum weight is smaller than 1. 
+
+**Actions:**
+
+* Check if the funds with higher impact are available in the fund_data file.
+* If they are not, try to get the corresponding fund data from LIM and add the data to the fund_data table before the next run.
+
+
+```{r}
+funds_coverage <- result_data_qa %>% 
+  filter(direct_holding == F &
+           asset_type == "Funds" &
+           investor_name != "Meta Investor") %>% 
+  select(fund_isin, currency,#isin, company_name,
+         value_usd, market_value) %>% 
+  group_by(fund_isin, currency) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T)),
+            sum_market_value = round(sum(market_value, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(
+    share_missing_value = round(sum_value_usd/
+      (investors_overview %>% filter(investor_name=="Meta Investor") %>% pull(sum_value_usd)), 5)
+    ) %>% 
+  arrange(fund_isin)
+datatable(funds_coverage)
+
+funds_coverage_missing_value <- result_data_qa %>% 
+  filter(isin == "MissingValue" &
+           investor_name != "Meta Investor") %>% 
+  select(fund_isin, currency,#isin, company_name,
+         value_usd, market_value) %>% 
+  group_by(fund_isin, currency) %>% 
+  summarise(sum_value_usd = round(sum(value_usd, na.rm=T)),
+            sum_market_value = round(sum(market_value, na.rm=T))) %>% 
+  ungroup() %>% 
+  mutate(
+    share_missing_value = round(sum_value_usd/
+      (investors_overview %>% filter(investor_name=="Meta Investor") %>% pull(sum_value_usd)), 5)
+    ) %>%  
+  arrange(fund_isin)
+datatable(funds_coverage_missing_value)
+
+
+```
+
+
+## Value by asset type
+**OPEN:**
+
+Target and Actions unclear
+One idea seems to be to check to share of sovereign bonds...
```

Comment:
Yep, I believe this is mainly about S.bonds. Laura has an additional analysis about these. And sometimes they need to be identified as a separate asset type.